mx.sulong/testsuites.py

```diff
@@ -61,8 +69,25 @@ def compileSuite(args=None):
     for testSuiteName in parsedArgs.suite:
         if parsedArgs.verbose:
             print('Compiling', testSuiteName, 'test suite')
-        command = testSuites[testSuiteName]
-        command()
+        compileCommand, _ = testSuites[testSuiteName]
+        compileCommand()
+
+def travisRunSuite(args=None):
+    """executes all the test suites or selected ones (see -h or --help)"""
+    vmArgs, otherArgs = mx_sulong.truffle_extract_VM_args(args)
+    parser = argparse.ArgumentParser(description="Compiles all or selected test suites.")
+    parser.add_argument('suite', nargs='*', help=' '.join(testSuites.keys()), default=testSuites.keys())
+    parsedArgs = parser.parse_args(otherArgs)
+
+    tasks = []
+    with mx_gate.Task('BuildJavaWithJavac', tasks) as t:
```

Comment:
I think that we should use the `mx gate` command for this in the future. What I did not know when I implemented the `su-travis*` command is that you can also select jobs through arguments.